src/utils/gadm_gpkg_processing.py

```diff
@@ -0,0 +1,200 @@
+#############################################################################
```

Comment:
I think this file might be better in the `transform` directory (perhaps with a bit more organization of that directory). The `utils` directory is intended for methods that can be used by multiple extraction / transform steps, while the ones in thie file are limited to GADM admin regions

Reply:
I hope that some of the methods in this file can be used with the extract of any GADM country so in that sense the intention was for it multiple use, albeit limited to GADM related actions. A GADM sub directory in Transform though does sound like an idea though.

Reply:
Ah good point, the first method in this file is in fact for the extraction. What if you moved that method to the `extract/gadm.py` file? Then I think `gadm_gkpg_processing.py` could be moved to `transform`.